pyproject.toml

```diff
@@ -26,3 +26,20 @@ line-length = 79
 [build-system]
 requires = ["poetry>=0.12"]
 build-backend = "poetry.masonry.api"
+
+[pycalver]
+current_version = "v202009.0001-alpha"
+version_pattern = "{pycalver}"
+commit = true
+tag = true
+push = true
+
+[pycalver.file_patterns]
+"pyproject.toml" = [
+    'current_version = "{version}"',
+]
+"README.md" = [
+    "{version}",
+    "{pep440_version}",
+]
+
```

Comment:
We're not yet using `pycalver` on this repo. It's not a bad idea for the future, but save for another PR :)

Reply:
Ah, I did that without thinking cuz GitHub sent the reminder

Reply:
Hmm, you probably got sent a reminder for a PR in another repo :joy: